app/lib/l10n/app_de.arb

```diff
@@ -25,6 +25,8 @@
   "noSupportOpenInBrowser": "Es scheint so, als ob dein Account oder deine Schule keine Features dieser App direkt unterstützt! Stattdessen kannst du Lanis noch im Browser öffnen.",
   "noInternetConnection1": "Keine Verbindung! Geladene Daten sind noch aufrufbar!",
   "noInternetConnection2": "Keine Internetverbindung!",
+  "lanisDownError": "Lanis ist nicht aufrufbar!",
+  "lanisDownErrorMessage": "Anscheinend ist Lanis nicht verfügbar da Lanis mit code 503 geantwortet hat.\nBitte überprüfe den Status von Lanis (PaedOrg) auf der Website.",
```

Comment:
The actual user (some 7th grader or even some teachers) do not have any idea what this means. Focus on the relevant information for them. "**Lanis is down**"

Reply:
Yeah i don't thought of that. I now removed the thing with the error code so it just says that lanis is down an they should check the status of Lanis (PaedOrg) on the status website